src/tasks/leveling.ts

```diff
@@ -1192,7 +1250,7 @@ export const LevelingQuest: Quest = {
     {
       name: "Powerlevel",
       completed: () =>
-        myBasestat($stat`Mysticality`) >= targetBaseMyst - targetBaseMystGap &&
+        myBasestat(myPrimestat()) >= targetBaseMyst - targetBaseMystGap &&
```

Comment:
Rename `targetBaseMyst` and `targetBaseMystGap` to a more general name like `targetBaseMainStat`. Also update the 2 preferences it refers to (and finally, add something to ensure that the old preferences are copied over to the new ones)

Reply:
Check my logic in Main, because I'm not entirely sure it will work.

Reply:
```ts
if (get("instant_targetBaseMyst", false)) {
    if (!get("instant_targetBaseMainStat", false)) set("instant_targetBaseMainStat", get("instant_targetBaseMyst"));    
    remove_property("instant_targetBaseMyst");
}
```

Reply:
Resolved.